jobs/JGLOBAL_LANDENS_ANALYSIS

```diff
@@ -0,0 +1,48 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+export WIPE_DATA="NO"
+export DATA=${DATA:-${DATAROOT}/${RUN}landensanl_${cyc}}
+source "${HOMEgfs}/ush/jjob_header.sh" -e "landensanl" -c "base landensanl"
+
+##############################################
+# Set variables used in the script
+##############################################
+# Ignore possible spelling error (nothing is misspelled)
+# shellcheck disable=SC2153
+GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
+gPDY=${GDATE:0:8}
+gcyc=${GDATE:8:2}
+GDUMP="enkfgdas"
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+# Generate COM variables from templates
+YMD=${PDY} HH=${cyc} generate_com -rx COM_LANDENS_ANALYSIS
```

Comment:
`COM_LANDENS_ANALYSIS_TMPL` is not defined anywhere in order to build this variable. Should this be
```suggestion
YMD=${PDY} HH=${cyc} generate_com -rx COM_LAND_ANALYSIS
```
(`COM_LAND_ANALYSIS` is already defined in `config.com`)

I also don't see a MEMDIR definition, which this will need if it is part of RUN=enkf. What `RUN`(s) is this part of, and if it is part of enkf, is it trying to write stuff for a bunch of different members (either as a metatask or looping over members within a single job)?